core/components/molecules/sidesheet/__stories__/scrolling.story.jsx

```diff
@@ -0,0 +1,230 @@
+import * as React from 'react';
+import { Sidesheet, Button, Column, Row, Label, Input, InputMask, Utils, Radio, Textarea, Slider } from '@/index';
+
+export const Scrolling = () => {
+  const [open, setOpen] = React.useState(false);
+  const dateMask = Utils.masks.date['mm/dd/yyyy'];
+  const dateValidator = (val) => Utils.validators.date(val, 'mm/dd/yyyy');
+
+  const onClose = () => {
+    setOpen(!open);
+  };
+
+  const headerOptions = {
+    heading: 'Goal details',
+    subHeading: 'Comorbidities (e.g obesity, chronic rhinosinusitis, food allergy)',
+  };
+
+  const options = {
+    onClose,
+    open,
+    headerOptions,
+    seperator: true,
+    footer: (
+      <>
+        <Button appearance="primary" className="mr-4">
+          Add goal
+        </Button>
+        <Button appearance="basic">Cancel</Button>
+      </>
+    ),
+  };
+
+  return (
+    <div>
+      <Button appearance="primary" onClick={() => setOpen(true)}>
+        Open Sidesheet
+      </Button>
+      <Sidesheet {...options}>
+        <Column className="py-7">
+          <Row>
+            <div className="w-100">
```

Comment:
Please follow the HTML semantics properly. `Column` is always used inside `Row` element

Reply:
Done